app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-video-interview-blog.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
```

Comment:
should this promo be deleted if we are not including blog promos in this PR?

Reply:
yep, on it